index.js

```diff
@@ -77,15 +79,25 @@ function storeOptionValue(parseOptions, option, value, result) {
   }
 }
 
-const parseArgs = (
+const parseArgs = ({
   argv = getMainArgs(),
   options = {}
-) => {
+} = {}) => {
   validateArray(argv, 'argv');
   validateObject(options, 'options');
-  for (const key of ['withValue', 'multiples']) {
-    if (ObjectHasOwn(options, key)) {
-      validateArray(options[key], `options.${key}`);
+  for (const [option, optionConfig] of ObjectEntries(options)) {
```

Comment:
Same comment about using `option` to index into `options` to get an `optionConfig`. (Disclaimer: I have not tried this at scale to check it helps, so feel free to question the concept and I'll give it a try!.)

Reply:
I'm all for it! https://github.com/pkgjs/parseargs/pull/63#discussion_r816256277

Reply:
Woop!